sockeye/training.py

```diff
@@ -340,7 +371,7 @@ def _fit(self,
         :param lr_decay_opt_states_reset: How to reset optimizer states after learning rate decay.
         """
         # TODO: Push update to MXNet to expose the optimizer (Module should have a get_optimizer method)
```

Comment:
can this TODO be moved up to the new _get_optimizer() method?

Reply:
makes sense.